opencga-analysis/src/main/java/org/opencb/opencga/analysis/tools/OpenCgaTool.java

```diff
@@ -308,6 +314,7 @@ private void privateCheck() throws Exception {
         if (toolParams != null) {
             toolParams.updateParams(getParams());
         }
+        check();
```

Comment:
The `check()` must be executed BEFORE the actual `run()` execution, why are you moving this call?

Reply:
This is executed before the actual `run()`. However, in the main code there should only be one call to `check` before running anything which is this `privateCheck()` method. This method will execute any common private checks and then call to the custom checks implemented by the tools. I just moved it from one place to another, but the sequence has not changed.

Reply:
Oh, I see. The diff made me think that this was still on the same method. My bad